### ISSUE TITLE
Update Qt examples to Qt6

### ISSUE THREAD

**Issue by kistlin** [arximboldi/lager#124]
As Qt6 is out, the examples could be updated to the newer version.

The following blog posts describe best practices.

https://www.qt.io/blog/introduction-to-the-qml-cmake-api
https://www.qt.io/blog/qml-modules-in-qt-6.2

A standalone setup of snake and the todo app using Qt6 can be found [here](https://github.com/kistlin/lager_qt6_standalone).
I was trying to setup Nix to directly do it in lager, but that didn't work and therefore did it as standalone version.

**Comment by arximboldi** [comment]
Yes, sadly Qt6 is not yet in nixpkgs, so getting those examples in the repo is a bit tricky at the moment: https://github.com/NixOS/nixpkgs/issues/108008

**Comment by kistlin** [comment]
Well I wasn't even aware of that, thank you. I had other problems with Nix :). I guess we'll wait then.

**Comment by kistlin** [comment]
Qt 6.3 should be available in nixpkgs by now.
[NixOS/nixpkgs#169296](https://github.com/NixOS/nixpkgs/pull/169296)

**Comment by arximboldi** [comment]
Yes. Even though no MacOS support yet. We could maybe make it conditional, not sure.
